### ISSUE TITLE
How to get file size ? 

### ISSUE THREAD

**Issue by abdurrahmanekr** [luisfuertes/react-native-file-picker#8]
default response : 
```javascript
{
	path : "/sdcard/Download/file.txt",
	uri : "file:///sdcard/Download/file.txt"
}
```
**imagePicker** default response:
```javascript
{
	data : "/9j/4QFRRXhpZg...",
	fileName : "file.txt",
	fileSize : 2353,
	height : 480,
	width : 480,
	isVertical : true,
	originalRotation : 0,
	path : "/sdcard/Download/file.txt",
	timestamp : "2017-04-25T05:07:43Z",
	type : "image/jpeg",
	uri : "file:///sdcard/Download/file.txt",
}
```

**Comment by abdurrahmanekr** [comment]
@yusufcakal can you help?

**Comment by yusufcakal** [comment]
@abdurrahmanekr what is this ?

**Comment by rhwish** [comment]
@yusufcakal  you can add some code in _FilePickerModule.java_ on _onActivityResult_ method like this `Activity currentActivity = getCurrentActivity();
      Uri uri = data.getData();
      // return uri
      response.putString("uri", data.getData().toString());
      // get path
      String path = null;
      path = getPath(currentActivity, uri);
      // get file size by path
      File f = new File(path);
      double size = f.length();
      // return file size
      response.putDouble("size",size);
      // return file name
      response.putString("name",uri.getLastPathSegment());`
may you success

**Comment by luisfuertes** [comment]
Fixed on v0.0.9, can you confirm it works?
